### PR TITLE
feat: More German language and a few other additions

### PR DESCRIPTION
**Pull request by mprangenberg**
- Multi-language support and translation for colony, contact, feral, job, journey, legend, mishap, module, monster, mutation, myzpower
- drawinit: Show discarded cards from haste for a better user experience. Otherwise you just see the result, without knowing how bad you could have had it :D
- optimized csv-import a little for standard-syntax (i saw that in one of the recently discarded pull requests)
- corrected some typos

### CODE REVIEW

File: generators/MYZMonsterGenerator.js [hunk 1]
```diff
@@ -83,12 +87,12 @@ class MYZMonsterGenerator extends YZGenerator {
 		 * The creature's type verbose description.
 		 * @type {string}
 		 */
-		this.descriptions.type = typeElem[0];
+		this.descriptions.type = this.qty > 1 ? typeElem[1] : typeElem[0];	// Return different Value for singular or plural monstertype, as some languages have different plural-endings.
```
Comment (Stefouch) [inline generators/MYZMonsterGenerator.js:90]: OK, great!

File: lang/locales.js [hunk 2]
```diff
@@ -5,7 +5,14 @@ const LOCALES = {
 	de: require('./de.json'),
 };
 
+/**
+ * Returns a locale entry (translated text)
+ * @param {string} text The key to look for in the translation table
+ * @param {string} locale The language code to use. Default is en (english)
+ */
 module.exports.__ = (text, locale) => {
+	if (typeof text != 'string') return text;
```
Comment (Stefouch) [inline lang/locales.js:14]:
```suggestion
	if (typeof text !== 'string') return text;
```

File: utils/Util.js [hunk 3]
```diff
@@ -556,6 +556,11 @@ class Util {
 				// else if (value === 'true') value = true;
 				// else if (value === 'false') value = false;
 				// else if (this.isNumber(value)) value = Number(value);
+				
+				// remove quotation marks, if someone uses common csv-syntax for text columns	// TODO (maybe): don't split on a separator within quotation marked text
+				if (value.startsWith('"') || value.startsWith("'")) value = value.substring(1);
```
Comment (Stefouch) [inline utils/Util.js:561]:
```suggestion
				if (value.startsWith('"') || value.startsWith("'")) value = value.slice(1);
```

File: utils/Util.js [hunk 4]
```diff
@@ -556,6 +556,11 @@ class Util {
...
+				if (value.endsWith('"') || value.endsWith("'")) value = value.substring(0, value.length - 1);
```
Comment (Stefouch) [inline utils/Util.js:562]:
```suggestion
				if (value.endsWith('"') || value.endsWith("'")) value = value.slice(0, -1);
```

File: yearzero/YZJourney.js [hunk 5]
```diff
@@ -15,11 +15,14 @@ class YZJourney {
 	 * @param {string} options.quarterDay The current Quarter of the Day (default is DAY), in UPPERCASE!
 	 * @param {string|string[]} options.terrains Terrain types, in UPPERCASE!
 	 * @param {boolean} options.fbr Whether to use Bitter Reach Mishaps
+	 * @param {string} options.lang The language code to use (default is 'en')
 	 */
 	constructor(filename, options = {}) {
 		this.filename = filename;
 		this.data = YZGenerator3.parse(filename);
 
+		this.lang = !options.lang ? 'en' : options.lang;
```
Comment (Stefouch) [inline yearzero/YZJourney.js:24]:
```suggestion
		this.lang = options.lang || 'en';
```

**Review by Stefouch — CHANGES_REQUESTED** [review]
Hi @mprangenberg,
Again, awesome work! Thank you also for having spotted and corrected English typos too!
Just a few minor changes left to do before merging!

File: commands/drawinit.js [hunk 6]
```diff
@@ -79,4 +82,9 @@ module.exports = {
 function getDrawCardText(cards, ctx, lang) {
 	if (!Array.isArray(cards)) return getDrawCardText([cards], ctx, lang);
 	return `${getMention(ctx.member)} **${__('initiative', lang)}:** ${cards.map(c => CARDS_ICONS[c]).join(' ')}`;
+}
+
```
[the remark this answers is not in the document]
Comment (mprangenberg) [reply commands/drawinit.js:86]: Yes, thanks for reminding me. I'm trying to get better with my documentation and commenting :D

File: lang/locales.js (hunk 2 above)
Comment (mprangenberg) [reply lang/locales.js:14]: Thanks for the suggestion. I'm still relatively new to JavaScript, coming from FoxPro and C# .Net...

**Review by Stefouch — APPROVED** [review]

File: lang/locales.js (hunk 2 above)
Comment (Stefouch) [reply lang/locales.js]: No worry, I'm not a programmer myself. JavaScript is just a hobby for me!

Most of the cases, you want to use `===` or `!==`.
The only exception I do, it's when I want to check for a **null** and an **undefined** value: a single `== null` is valid for both.

**Pull request merged**